.github/workflows/automatic-reply.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow helps to automate replying to newly created issues and PRs in a repository.
+# Standard comment templates are set up for that.
+# The workflow offers a convenient way to set up your own comment templates for both issues and PRs using the following inputs:
+# - issue-comment-template
+# - pr-comment-template
+# These inputs support calling for github actions contexts.
+#
+# It's possible to toggle automatic replying on and off for issues or pull requests, or for both of them using workflow triggers such as:
+#
+# on:
+#  issues:
+#    types: [opened]
+#  pull_request:
+#    types: [opened]
+
+
+
+name: Automatic reply to new issues and PRs
+
+on:
+  workflow_call:
+    inputs:
+        issue-comment-template:
+          description: "Optional input to set up a comment template for newly created issues"
+          required: false
+          type: string
+          default: "Hi @${{github.event.sender.login}} :wave:! Thanks for the issue, it's now tracked by our team of maintainers. As soon as an investigation is done, they'll get back to you with updates."
+        pr-comment-template:
+          description: "Optional input to set up a comment template for newly created PRs"
+          required: false
+          type: string
+          default: "Hi @${{github.event.sender.login}} :wave:! Thanks for the PR, it's now tracked by our team of maintainers. As soon as an investigation is done, they'll get back to you with feedback."
+
+jobs:
+  automatic-reply:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get issue/PR number
+      id: get-information
+      run: |
+        $EntityNumber = ("${{github.event_name}}" -ceq "issues" ) ? "${{github.event.issue.number}}" : "${{github.event.pull_request.number}}"
+        echo "ENTITY_NUMBER=$($EntityNumber)" >> $Env:GITHUB_OUTPUT
+      shell: pwsh
```

Comment:
You can specify the [default shell](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun) for the entire job